design/src/rfcs/rfc0023_request_id.md

```diff
@@ -0,0 +1,103 @@
+RFC: RequestID in business logic handlers
+=============
+
+> Status: RFC
```

Comment:
Does this apply to clients, servers, or both?

Reply:
Servers only. I'll make it clear in the RFC